python_modules/dagster/dagster/_core/definitions/graph_definition.py

```diff
@@ -795,62 +797,54 @@ def is_subselected(self) -> bool:
 
 def _validate_in_mappings(
     input_mappings: Sequence[InputMapping],
-    solid_dict: Mapping[str, Node],
+    nodes_by_name: Mapping[str, Node],
     dependency_structure: DependencyStructure,
     name: str,
     class_name: str,
-) -> Tuple[Sequence[InputMapping], Sequence[InputDefinition]]:
+) -> Sequence[InputDefinition]:
     from .composition import MappedInputPlaceholder
 
-    input_def_dict: Dict[str, InputDefinition] = OrderedDict()
+    input_defs_by_name: Dict[str, InputDefinition] = OrderedDict()
     mapping_keys = set()
 
     for mapping in input_mappings:
         # handle incorrect objects passed in as mappings
```

Comment:
not sure these error checks make sense with new apis but not really critical to clean them up